src/PhpSpec/Listener/NamedConstructorNotFoundListener.php

```diff
@@ -79,6 +79,17 @@ public function afterSuite(SuiteEvent $event)
                     'arguments' => $arguments
                 ));
                 $event->markAsWorthRerunning();
+
+                if (!method_exists($classname, '__construct')) {
```

Comment:
It'd be a different message, and a different generator... it just seemed like an edge case to me?

By the same token, could be added in future.